src/styles/index.scss

```diff
@@ -1,3 +1,127 @@
+$main-accent: #060b35;
+$secondary: #616070;
+
 body {
   margin: 0;
 }
+
+.block {
+  width: 200px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: solid 1px #f3f3f3;
+  border-radius: 5px;
+
+  &:hover{
+    .block__link{
+      background-color: #fff;
+      color: #00acdc;
+      border: solid 1px #00acdc;
+    }
+  }
+  &__img {
+    background-image: url('../images/imac.jpeg');
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__text {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-accent;
+    padding-top: 40px;
+  }
+
+  &__under-text {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $secondary;
+    padding-top: 4px;
+    align-self: flex-start;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    padding-top: 16px;
+    align-items: center;
+
+    &-stars {
+      display: flex;
+    }
+  }
+
+  &__review-text {
+    font-weight: 400;
+    font-size: 10px;
+    color: $main-accent;
+  }
+
+  &__price {
+    padding-top: 24px;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price-text {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary;
+    padding: 0;
+  }
+
+  &__price-number {
+    padding: 0;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-accent;
+  }
+
+  &__link {
+    margin-top: 16px;
+    width: 166px;
+    height: 40px;
+    border-radius: 5px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    color: #fff;
+    background-color: #00acdc;
+    border: 1px solid #00acdc;
+    text-transform: uppercase;
+    display: flex;
+    text-decoration: none;
+    align-items: center;
+    justify-content: center;
+
+  }
+}
+
+
+.stars {
```

Comment:
Create a separate file for each bem block

Reply:
Hi, I'm getting an error on import